### ISSUE TITLE
Distinctly-styled nodes, by type

### ISSUE THREAD

**Issue by maxkfranz** [PathwayCommons/factoid#682]
TODO

## Description

**Q: What is the name of the feature?**

A: 

**Q: What does this feature enable the user to do?**

A: 

**Q: What information must the user provide to use the feature?**

A: 

**Q: What are the applicable constraints, e.g. compatibility or performance?**

A: 

**Q: How does this feature affect each class of user (persona)?**

A:

- Biologist: 
- Editor: 
- Computational biologist: 
- Curator:

**Comment by maxkfranz** [comment]
A chemical node should look something like this:

<img width="289" alt="Screen Shot 2021-01-04 at 11 14 44 AM" src="https://user-images.githubusercontent.com/989043/103555357-22b40100-4e7e-11eb-896e-affefd053dc4.png">

A protein is usually shown as something like the following, but we should probably use something simpler:

<img width="349" alt="Screen Shot 2021-01-04 at 11 17 27 AM" src="https://user-images.githubusercontent.com/989043/103555660-99e99500-4e7e-11eb-8955-131d291d4fac.png">

**Comment by jvwong** [comment]
FYI: https://pubs.rsc.org/en/content/articlelanding/2021/cb/d0cb00197j

![d0cb00197j-f2_hi-res](https://user-images.githubusercontent.com/4706307/103794982-6a1fc600-5013-11eb-8fbc-aad8a09a007f.gif)

**Comment by maxkfranz** [comment]
Yeah, the simplest version of those kind of icons would be just circles and hexagons. 

Our complexes don't look like a typical figure, but there are tradeoffs re. editing

**Comment by jvwong** [comment]
An example of where this would help [PDCD2 functions as an evolutionarily conserved chaperone dedicated for the 40S ribosomal protein uS5 (RPS2). Anne-Marie Landry-Voyer et al, Nucleic Acids Res 48 2020](https://biofactoid.org/document/047c96a4-1006-4052-aa97-9e42ed8e6a9e).  Note the groundings are partially incorrect

<img width="1328" alt="Screen Shot 2021-09-20 at 10 50 28 AM" src="https://user-images.githubusercontent.com/4706307/134023493-5f822934-4325-4848-8d21-4a8eb04283d7.png">
